addons/website_project/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Deprecated, it shouldn't be added anymore, same for the file below. You can also remove it from the files you modified (but only those) as it will not impact the blame neither on the file nor on the lines :)
```suggestion
```